bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanConfig.java

```diff
@@ -58,16 +61,7 @@ public void setMacAddress(String macAddress) {
     }
 
     public boolean isValid() {
-        if (label == null || label.isBlank()) {
-            return false;
-        }
-        if (ipAddress == null || ipAddress.isBlank()) {
-            return false;
-        }
-        if (macAddress == null || macAddress.isBlank()) {
-            return false;
-        }
-        return true;
+        return !(label.isBlank() || ipAddress.isBlank() || macAddress.isBlank());
```

Comment:
Just for consideration - might be simpler to read (from left to right without two passes):
```suggestion
        return !label.isBlank() && !ipAddress.isBlank() && !macAddress.isBlank());
```